packages/output/src/utils/styles.js

```diff
@@ -49,6 +49,8 @@ function CustomStyles() {
       dangerouslySetInnerHTML={{
         __html:
           `
+              h1, h2, h3 { font-weight: normal; }
```

Comment:
@swissspidy 

1) is this the correct spot to add these styles
2) should I see these classes applied if I `npm run build` or is some other step required 

Trying to gain some understanding as to how this file works for future reference

Reply:
Yes this is the correct spot.

These styles are used for generating the AMP output that you see when previewing the story on the frontend.

To test that it works correctly, follow the reproduction steps from the ticket itself.